8Knot/db_manager/augur_manager.py

```diff
@@ -161,55 +174,70 @@ def run_query(self, query_string: str) -> pd.DataFrame:
     def multiselect_startup(self):
         logging.warning(f"MULTISELECT_STARTUP")
 
-        query_string = f"""SELECT DISTINCT
```

Comment:
Same with this section, if there are edits change it to only impact the lines with actual changes